src/index.css

```diff
@@ -1,27 +1,300 @@
+
```

Comment:
- [ ] As the last reviewer mentioned, please align the navbar elements to the rest of the ones below them to comply with the project requirements and to make it look similar to the design provided.
- [ ] Please also position your category filter droplist in the position of the "categories" text, replace the text with your filter element.

![image](https://user-images.githubusercontent.com/38522578/140295090-477e9908-9c22-428c-a701-7b08d27f9c9a.png)

Reply:
Hi @e71az 

I have made those changes already you are requesting, kindly have a look.

Reply:
Hello Tanzila, I didn't see them reflected in my last review, sorry.

Now I can actually see the categories change that you did which is great, however, there is an issue with your navbar, depending on the zoom used the contents inside it get separated or together, I will attach some screenshots to better explain what I mean:

100% zoom:

![image](https://user-images.githubusercontent.com/38522578/140299401-aecb1022-b654-4ca9-b911-253441564d6d.png)

125% zoom:

![image](https://user-images.githubusercontent.com/38522578/140299482-42de3cbb-e713-49ef-8d2b-a666cf3022b3.png)

150% zoom:

![image](https://user-images.githubusercontent.com/38522578/140299507-499a5851-13d5-423d-adb0-2c3a760ff638.png)

175% zoom:

![image](https://user-images.githubusercontent.com/38522578/140299547-ca3c8865-e21a-4988-aa72-debad27513d1.png)

In order to see the navbar as we are expecting it, I have to go as far as 175% zoom, notice that the navbar elements are the only ones changing position, the rest of your page is static which is great for this milestone

- [ ] Please fix this small detail to get the project approved, good luck 👍